docker-compose.yaml

```diff
@@ -13,19 +13,22 @@ services:
             - DD_APM_NON_LOCAL_TRAFFIC=true
             - DD_API_KEY=${DD_API_KEY}
             - DD_SITE=${DD_SITE:-datadoghq.com}
-            - DD_ENV=prod
-    default:
+            - DD_ENV=dev
```

Comment:
Do we want to have differences for open source usage dev and then internal datadog usage dev?

Reply:
I don't think so.